tox.ini

```diff
@@ -0,0 +1,55 @@
+[tox]
+envlist =
+    py,
+    doc,
+    sty,
+passenv = PYTHONPATH
+skip_install = True
+skip_missing_interpreters = True
+skipsdist = True
+
+[testenv]
+description =
+    Run all unit tests
+allowlist_externals =
+    pytest
+commands =
+    pytest {posargs}
+
+[testenv:doc]
+description =
+    Build documentation
+changedir = docs
+allowlist_externals =
+    make
+commands =
+    make html
+
+[testenv:doclive]
```

Comment:
Would using tox then be the best way to start rebuilding the docs? If yes then we should also update https://mpl-interactions.readthedocs.io/en/stable/Contributing.html

Reply:
Yeah sorry, it's only later that I notice the `make watch` you defined
https://github.com/ianhi/mpl-interactions/blob/2eeda96afe6d2d9e6c7354818e7966f0f8a7ad43/docs/Makefile#L22-L23

I personally prefer tox though (it's basically a wrapper around  `make`), because you can run it from any directory in the repo. And in a way, you can use it as local CI.